src/libs/tools/src/plugindatabase.cpp

```diff
@@ -223,6 +223,21 @@ std::string ModulesPluginDatabase::lookupInfo (PluginSpec const & spec, std::str
 	return plugin->lookupInfo (which);
 }
 
+PluginDatabase::func_t ModulesPluginDatabase::getSymbol (PluginSpec const & spec, std::string const & which) const
+{
+	PluginPtr plugin;
+	try
+	{
+		plugin = impl->modules.load (spec.getName (), spec.getConfig ());
+	}
+	catch (...)
+	{
+		throw;
+	}
```

Comment:
@Namoshek I agree with you - it looks odd. The code was introduces in 10e15c4a2034805260dcae55d268745a7a2aa4b1 by @markus2330 .